README.md

```diff
@@ -1,65 +1,128 @@
 # Data Exporter (post-release)
 
-This program will produce data exports for submission to NCBI, UCSC, and Europe PMC.  Files provided to these resources are:
-* **NCBI:** Gene (NCBI gene identifier to Reactome top level pathways) and Protein (all UniProt entries in Reactome associated with any NCBI Gene identifier).
-* **UCSC:** Entity (UniProt entries in Reactome) and Events (UniProt entries in Reactome in relation to Reactome pathways and reactions).
-* **Europe PMC:** Profile and Link XML (describing Reactome as a provider and Reactome Pathway to literature references, respectively).
-
-The file outputs will be as follows in the configured output directory (see configuration section below) and where XX is the Reactome Version:
-
-* gene_reactomeXX-Y.xml (where Y is the file number as this file is split between multiple files so the file upload size is acceptable for NCBI)
+This program will produce data exports for submission to NCBI, UCSC, and Europe PMC.  Files provided to these resources
+are:
+* **NCBI:** Gene (NCBI gene identifier to Reactome top level pathways) and Protein (all UniProt entries in Reactome
+ associated with any NCBI Gene identifier).
+* **UCSC:** Entity (UniProt entries in Reactome) and Events (UniProt entries in Reactome in relation to Reactome 
+pathways and reactions).
+* **Europe PMC:** Profile and Link XML (describing Reactome as a provider and Reactome Pathway to literature 
+references, respectively).
+
+The file outputs will be as follows in the configured output directory (see configuration section below) and where XX 
+is the Reactome Version:
+
+* gene_reactomeXX-Y.xml (where Y is the file number as this file is split between multiple files so the file upload 
+size is acceptable for NCBI)
 * proteins_versionXX (local file that is NOT uploaded to NCBI)
 * protein_reactomeXX.ft
 * ucsc_entityXX
 * ucsc_eventsXX
 * europe_pmc_profile_reactome_XX.xml
 * europe_pmc_links_reactome_XX.xml
 
+After the files have been generated, the following files will be uploaded to external FTP Servers:
+
+* To the **NCBI FTP Server**, the "gene_reactomeXX-Y.xml" and "protein_reactomeXX.ft" files will be uploaded (using the 
+values provided in the configuration file to connect to the FTP Server - see the [Configuration](#configuration) 
+section below)
+
+* To the **EuropePMC FTP Server**, the "europe_pmc_profile_reactome_XX.xml" and "europe_pmc_links_reactome_XX.xml" 
+files will be uploaded (using the values provided in the configuration file to connect to the FTP Server - see the 
+[Configuration](#configuration) section below)
```

Comment:
Similarly to the above comment, a linkout to these FTP servers where the Reactome-produced files would appear?

Reply:
Yup, I will do that.  They both require credentials to access, but I will mention that in the README.

Reply:
I tried to add links to the FTP servers in two different ways in the README, but it looks like GitHub will not recognize a URL with an FTP protocol in their markdown:

https://github.com/gollum/gollum/issues/759

The links to the FTP servers are still in the README if GitHub does resolve this issue at some point.

Reply:
Sounds good, thanks for attempting it!